### ISSUE TITLE
Special Character in Name not correctly displayed in multiple Menus.

### ISSUE THREAD

**Issue by CodeAvolition** [Digitaler-Impfnachweis/covpass-ios#89]
## Avoid duplicates
* [ ] Bug is not mentioned in the [FAQ](https://digitaler-impfnachweis-app.de/faq/)
* [ ] Bug is not already reported in another issue

## Technical details

- Device name:
- Iphone X
- OS version:
- IOS 15.1
- App version:
- 1.13

## Describe the bug
- Name with the special Character `ë` is displayed as `‰` on the main screen where the name is displayed under the certificate.
- The same thing happens in the personal Details menu.
- in Standardized Name the hyphen and ë are also not correctly displayed.

- This is an issue because when it gets checked on public non technicall people think the name is wrong and don't understand why.
## Steps to reproduce the issue
Have an ë in your name.

![IMG-20211231-WA0010__01__01](https://user-images.githubusercontent.com/65223214/147828759-c0cdf451-3d1e-4280-b377-703047c9324a.jpg)

## Expected behaviour
The name should be displayed as [...] Joël [...]

## Possible Fix
Correctly displayed special characters that might not be in the expected set of characters?

## Additional context

- this is a problem because i already got into multiple arguments with people not believing the certificate to be mine because of the issues with the name.

**Comment by Ein-Tim** [comment]
1. The QR-Code can be used to import your certificate
2. It seems like the issuer of the certificate entered your name wrong. This does not seem like a bug in the app. Maybe you can get a new certificate issued?

**Comment by timokoenig** [comment]
This character is part of the UTF8 character set so nothing special IMO.

This is most likely an issue of the certificate especially if the standardized name is also wrong. You will need to get a new certificate in hopes that the system where this certificate gets created will correctly format your name.

**Comment by Kurt57** [comment]
As you already figured out its nothing special for the App. Seems likely an issue of the certificate.

**Comment by timokoenig** [comment]
That is correct but has the team done anything to mitigate the issue? e.g. talk to the people that build the web frontend? Doing nothing is not the right thing when you actually care about building a great product.

**Comment by Kurt57** [comment]
If you're thinking of the overall project, you're absolutely right. But this is the iOS App Repository and its context. I just find it more efficient to properly address the issues. I'll take the issue with me anyway.

**Comment by timokoenig** [comment]
Well then moving the ticket in the right repository and talking to the right people would be a better option than closing a ticket

**Comment by CodeAvolition** [comment]
i agree with the statements made that it would be beneficial to get this issue to the people able to do sth. about it, but i dont know the proper channels to do that. if this is not the right place for the issue than sure close it as you see fit.
